examples/oauth/app/auth.server.ts

```diff
@@ -0,0 +1,76 @@
+import { createCookieSessionStorage } from 'remix'
+import { Authenticator, AuthorizationError } from 'remix-auth'
+import { SupabaseStrategy } from 'remix-auth-supabase'
+import { supabaseAdmin } from '~/supabase.server'
+import type { Session } from '~/supabase.server'
+
+export const sessionStorage = createCookieSessionStorage({
+  cookie: {
+    name: 'sb',
+    httpOnly: true,
+    path: '/',
+    sameSite: 'lax',
+    secrets: ['s3cr3t'], // This should be an env variable
+    secure: process.env.NODE_ENV === 'production',
+  },
+})
+
+export const supabaseStrategy = new SupabaseStrategy(
```

Comment:
let's remove this one (as it has it's own dedicated example already :)

Reply:
Done :)